### PR TITLE
Updates to the Alerting UI section

### PR DESCRIPTION
**Pull request by rh-max**

### CODE REVIEW

File: modules/monitoring-contents-of-the-alerting-ui.adoc [hunk 1]
```diff
@@ -33,4 +33,17 @@ image::silences-screen.png[]
...
-Additionally, next to the title of each of these pages is a link to the old Alertmanager interface.
+The *YAML* page is located in *Monitoring* -> *Alerting* -> *YAML* of the OpenShift Container Platform web console.
+
+image::yaml-screen.png[]
+
+. Upload a file with Alertmanager configuration.
+. Examine and edit the current Alertmanager configuration.
+. Save the updated Alertmanager configuration.
+
+Also, next to the title of each of these pages is a link to the old Alertmanager interface.
```
Comment (dtaylor113) [inline modules/monitoring-contents-of-the-alerting-ui.adoc]: `..is a link to the old Alertmanager interface`   Not sure if 'old' is the best term here.   Maybe 'Promethus interface'?

Comment (rh-max) [reply modules/monitoring-contents-of-the-alerting-ui.adoc]: It's the term we agreed to use to imply that people should switch to the new OCP interface. And it's not the Prometheus interface anyway, the Prometheus one is linked from the Metrics page. This is the AM one.

**Review by dtaylor113 — APPROVED** [review]

**Comment by rh-max** [comment]
@sheriff-rh Could you please take a look at this? If it's good, please merge & cherry-pick to 4.2 too. Thanks.

**Comment by sheriff-rh** [comment]
Sure thing @rh-max !

**Comment by rh-max** [comment]
@sheriff-rh Thanks! Yeah, I should have said cherry-pick into master. How about just running `/cherrypick master`? The effect will be the same as opening a new PR against master & merging that.

**Comment by sheriff-rh** [comment]
> @sheriff-rh Thanks! Yeah, I should have said cherry-pick into master. How about just running `/cherrypick master`? The effect will be the same as opening a new PR against master & merging that.

Yep that got it, good call @rh-max !

**Comment by rh-max** [comment]
Thanks @sheriff-rh !

File: modules/monitoring-contents-of-the-alerting-ui.adoc [hunk 2]
```diff
@@ -7,7 +7,7 @@
 
 This section shows and explains the contents of the Alerting UI, a Web interface to the Alertmanager.
 
-The main two pages of the Alerting UI are the *Alerts* and the *Silences* pages.
+The main three pages of the Alerting UI are the *Alerts*, the *Silences*, and the *YAML* pages.
 
 The *Alerts* page is located in *Monitoring* -> *Alerts* of the {product-title} web console.
```
[the remark this answers is not in the document]
Comment (rh-max) [reply modules/monitoring-contents-of-the-alerting-ui.adoc]: Implemented in #16978.

**Pull request merged**